src/windows/Link.tsx

```diff
@@ -32,6 +35,12 @@ export interface LinkState {
 }
 
 export class Link extends LinkBase<LinkState> implements FocusManagerFocusableComponent {
+    
+    //Offset to show context menu using keyboard.
+    protected _contextMenuOffset() {
+        return {x: 0, y: 0};
```

Comment:
Nit: there should be extra spaces within brackets.

Reply:
updated